cam2/src/main/java/com/commonsware/cwac/cam2/ConfirmationFragment.java

```diff
@@ -49,6 +49,7 @@
   private ImageHelper imageHelper = new ImageHelper();
   private boolean facePass = false;
   private boolean sensorPass = false;
+  private boolean faceClosePass = false;
```

Comment:
we changed the behaviour to show any error only once. next time even if face is not present or close enough, we all the user to take the photo. This is because all these detections are not highly reliable. so we only make the user retake at max once for each type of error